test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -168,6 +168,7 @@ def test_dist_operations_fastai_gpu(framework_version, ecr_image, sagemaker_regi
 @pytest.mark.skip_cpu
 @pytest.mark.skip_py2_containers
 @pytest.mark.skip_trcomp_containers
+@pytest.mark.skip_pt20_cuda121_tests
```

Comment:
Why skip this SM Model Parallel test for PT 2.0 cu121 containers?

Reply:
as there are no SM model parallel binaries in image